tests/unit/Swift/Transport/MailTransportTest.php

```diff
@@ -391,7 +418,7 @@ public function testMessageHeadersOnlyHavePHPEolsDuringSending()
     }
 
     /**
-     * @expectedException Swift_TransportException
+     * @expectedException \Swift_TransportException
```

Comment:
Sorry, your fabbot.io told me to, I can revert but woould probably trigger a false check on fabbot, see the patch here: http://fabbot.io/patch/swiftmailer/swiftmailer/846/aa17624da761b7a12ffa46590be1f06ed7ae9c4a/cs.diff